### PR TITLE
chore: Use the original conf/config.yaml + sed to replace `api/test/docker/manager-api-conf.yaml`

### LINKED ISSUE #698: chore: use the original conf/config.yaml + sed to replace `api/test/docker/manager-api-conf.yaml`
refer to:
https://github.com/apache/apisix-dashboard/pull/693#issuecomment-722218703

### PR DESCRIPTION
**Pull request by Jaycean**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues
close #698 
___
### Chore
- use the original conf/config.yaml + sed to replace `api/test/docker/manager-api-conf.yaml`
- manage-api-conf.yaml Keep the local test for use, and then put the E2E test step into the makefile for execution, and then delete it.

### CODE REVIEW

**Comment by Jaycean** [comment]
ping @nic-chen @imjoey 
I'm not sure that every job uses conf.yaml And whether mutual modification has an impact. So I'm interested in deploy-with- docker.yaml More than one configuration has been deleted.

File: .github/workflows/deploy-with-docker.yml [hunk 1]
```diff
@@ -30,6 +30,8 @@ jobs:
         run: |
           sed -i 's/127.0.0.1:2379/172.16.238.10:2379/' api/conf/conf.yaml
           sed -i 's/host: 127.0.0.1/host: 0.0.0.0/' api/conf/conf.yaml
+          sed -i '/172.16.238.11:2379/d' api/conf/conf.yaml
+          sed -i '/172.16.238.12:2379/d' api/conf/conf.yaml
```
Comment (imjoey) [inline .github/workflows/deploy-with-docker.yml]: @Jaycean IMHO, there is no need to remove the two lines here, as each job will be running in independent environment and configurations will not be affected by others.

@nic-chen what's your opinions?

**Comment by nic-chen** [comment]
> ping @nic-chen @imjoey
> I'm not sure that every job uses conf.yaml And whether mutual modification has an impact. So I'm interested in deploy-with- docker.yaml More than one configuration has been deleted.

No, each CI is independent of each other

**Review by imjoey — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy-with-docker.yml (hunk 1 above)
Comment (nic-chen) [reply .github/workflows/deploy-with-docker.yml]: @imjoey  sorry, I don't get the point.. why we should change `deploy-with-docker.yml`, it's not related to this pr.

Comment (Jaycean) [reply .github/workflows/deploy-with-docker.yml]: @nic-chen This modification was added before(d07c1307ae75153d360749468091ef1e8e7bc514). @imjoey  didn't think it was necessary to add it, but it has been removed after discussing with you.

Comment (imjoey) [reply .github/workflows/deploy-with-docker.yml]: > @imjoey sorry, I don't get the point.. why we should change `deploy-with-docker.yml`, it's not related to this pr.

@nic-chen 😄 just a misunderstanding. We both consider `deploy-with-docker.yml` should not be changed.

File: docs/back-end-e2e.md [hunk 2]
```diff
@@ -78,6 +78,18 @@ This document describes how to use E2E test locally.
 
 1. [install docker-compose](https://docs.docker.com/compose/install/)
 
+**NOTE:** To run docker compose locally, you need to modify ./api/conf/conf.yaml 'host' and 'endpoints' configuration in the file
```
Comment (imjoey) [inline docs/back-end-e2e.md]: @Jaycean  How about here we say:  ( Thanks ) 


**NOTE:** In order to run docker compose locally, please change the values of `listen.host` and `etcd.endpoints` within `./api/conf/conf.yaml` as follows:

Comment (Jaycean) [reply docs/back-end-e2e.md]: Thks,Your translation is better than mine. Done.

Comment (imjoey) [reply docs/back-end-e2e.md]: > Thks, It‘s better than mine. Done.

@Jaycean thanks.

**Comment by imjoey** [comment]
The CI check `Code scanning results / CodeQL` suddenly failed due to the url regex `/^[0-9a-zA-Z]([-.\w]*[0-9a-zA-Z])*(:(0-9)*)*(\/?)([a-zA-Z0-9\-\=\.\"\?\,\'\/\\\+&amp;%$#_]*)?$/;` is an `Inefficient regular expression` at  file `web/src/pages/Route/components/DebugViews/DebugDrawView.tsx`, please see https://github.com/apache/apisix-dashboard/pull/1417/checks?check_run_id=1823285766 for details.

While it's not changed within this PR and even it was last updated at about 9 days ago in https://github.com/apache/apisix-dashboard/commit/b0c531d948dd62ed2e0393ffdf6b2e9fc7f4b15a authored by @liuxiran . Not sure whether it's 100% reproducible and need more test results. 

@nic-chen @liuxiran  do you have any ideas about this? Thanks.

**Comment by nic-chen** [comment]
> The CI check `Code scanning results / CodeQL` suddenly failed due to the url regex `/^[0-9a-zA-Z]([-.\w]*[0-9a-zA-Z])*(:(0-9)*)*(\/?)([a-zA-Z0-9\-\=\.\"\?\,\'\/\\\+&amp;%$#_]*)?$/;` is an `Inefficient regular expression` at file `web/src/pages/Route/components/DebugViews/DebugDrawView.tsx`, please see https://github.com/apache/apisix-dashboard/pull/1417/checks?check_run_id=1823285766 for details.
> 
> While it's not changed within this PR and even it was last updated at about 9 days ago in [b0c531d](https://github.com/apache/apisix-dashboard/commit/b0c531d948dd62ed2e0393ffdf6b2e9fc7f4b15a) authored by @liuxiran . Not sure whether it's 100% reproducible and need more test results.
> 
> @nic-chen @liuxiran do you have any ideas about this? Thanks.



We should determine whether this CI now fails every time, perhaps because the action library used has been changed? we could fix it in another PR.

**Review by starsz — COMMENTED** [review]
We can remove the file `manager-api-conf.yaml` right?

**Comment by Jaycean** [comment]
> We can remove the file `manager-api-conf.yaml` right?

After discussing this problem with @nic-chen , We plan not to delete it for the time being. We will submit a new PR, add the relevant steps of E2E test to the makefile, and then delete the file.

**Comment by starsz** [comment]
> > We can remove the file `manager-api-conf.yaml` right?
> 
> After discussing this problem with @nic-chen , We plan not to delete it for the time being. We will submit a new PR, add the relevant steps of E2E test to the makefile, and then delete the file.

Got it.

**Review by starsz — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

**Comment by nic-chen** [comment]
we could merge this PR first.

@liuxiran  please help check CI `Code scanning results / CodeQL` when you have time. Thanks.

**Comment by nic-chen** [comment]
ping @imjoey

**Review by imjoey — APPROVED** [review]
/lgtm

**Comment by starsz** [comment]
@imjoey Don't forget to delete useless commit info when merge commit.

**Comment by imjoey** [comment]
> @imjoey Don't forget to delete useless commit info when merge commit.

@nic-chen sorry for that. I did in Github iOS app and will pay more attention on this. Thanks for reminding. 😄

**Pull request merged**